UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample.Wasm/LinkerConfig.xml

```diff
@@ -1,6 +1,7 @@
 ﻿<linker>
 	<assembly fullname="Uno.UI">
-		<type fullname="*" />
+	<assembly fullname="Microsoft.Extensions.Options" />
+	<type fullname="*" />
```

Comment:
This is not valid.

Reply:
Agree. It was there before, and wasn't sure if would be required somewhere.  Removing.